staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -392,7 +411,10 @@ func (u *Unstructured) SetDeletionGracePeriodSeconds(deletionGracePeriodSeconds
 }
 
 func (u *Unstructured) GetLabels() map[string]string {
-	m, _, _ := NestedStringMap(u.Object, "metadata", "labels")
+	m, _, err := NestedStringMap(u.Object, "metadata", "labels")
+	if err != nil {
+		utilruntime.HandleError(err)
```

Comment:
don't change this accessor to call `utilruntime.HandleError`, that has the potential to fill server logs with lots of inactionable logs